pkg/controllers/statusaggregator/controller.go

```diff
@@ -131,6 +133,7 @@ func newStatusAggregator(controllerConfig *util.ControllerConfig,
 		typeConfig:    typeConfig,
 		plugin:        plugin,
 		metrics:       controllerConfig.Metrics,
+		logger:        klog.LoggerWithName(klog.Background(), ControllerName),
```

Comment:
The convention is not to use a named logger, but to inject the controller name as a key-value. `klog.LoggerWithValues(klog.Background(), "controller", ControllerName, "ftc", typeConfig.Name)`